app/controllers/lists_controller.rb

```diff
@@ -9,24 +9,22 @@ def index
   def show
     authorize @list
     @list_ingredients = @list.list_ingredients
+    @new_ingredient = Ingredient.new
   end
 
   def new
     @list = List.new
-    @fridge_ingredients = FridgeIngredient.find(params[:fridge_ingredients_id])
     authorize @list
   end
 
   def create
     @list = List.new(list_params)
     @list.user = current_user
-    @fridge_ingredients = FridgeIngredient.find(params[:fridge_ingredient_id])
-    @list.fridge_ingredient = @fridge_ingredient
 
     authorize @list
 
-    if @list.save!
-      redirect_to list_path(@list)
+    if @list.save
+      redirect_to lists_path
```

Comment:
are you trying to redirect to the list index? This goes for all other instances where you've used "listS_path"

Reply:
Hi Dom, this was done and checked by Vlad so all good :)